lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -197,7 +197,10 @@ def add_payment_method(doc, payment_method, options)
         if payment_method.is_a?(String)
           doc.token do
             doc.litleToken(payment_method)
-            doc.expDate(options[:exp_date]) if options[:exp_date]
+            if options[:exp_month] && options[:exp_year]
+              expiration_date = "#{format(options[:exp_month], :two_digits)}#{format(options[:exp_year], :two_digits)}"
```

Comment:
03e08a1

On Wed, Jul 6, 2016 at 9:26 AM, Fabio Kreusch <notifications@github.com>
wrote:

> In lib/active_merchant/billing/gateways/litle.rb
> <https://github.com/activemerchant/active_merchant/pull/2116#discussion_r69720248>
> :
>
> > @@ -197,7 +197,10 @@ def add_payment_method(doc, payment_method, options)
> >          if payment_method.is_a?(String)
> >            doc.token do
> >              doc.litleToken(payment_method)
> > -            doc.expDate(options[:exp_date]) if options[:exp_date]
> > +            if options[:exp_month] && options[:exp_year]
> > +              expiration_date = "#{format(options[:exp_month], :two_digits)}#{format(options[:exp_year], :two_digits)}"
>
> Maybe refactor the existing expiration_date method so that it can be used
> here too?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/activemerchant/active_merchant/pull/2116/files/877e0a4af24b3de30d6a3bc9c1ee5802e7c3f068..9449ae41640035b2cd95e533f7b51eb749892abf#r69720248>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJOzeG3Z_Xe05x3xpf0tqbf5sOyGIstks5qS57dgaJpZM4IorHs>
> .
>



-- 
Rubem Nakamura